Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## A quick demo of the Abstract Factory Pattern\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 8d14291a2d91698843200c9befb99d0352c5c13b)
+++ b/README.md	(date 1744836506737)
@@ -1,1 +1,4 @@
 ## A quick demo of the Abstract Factory Pattern
+# Instruction
+- Run Main.java for the demo
+
